src/math/auditory/mel_filterbank_16.c

```diff
@@ -74,9 +74,10 @@ void psy_apply_mel_filterbank_16(struct psy_mel_filterbank *fb, struct icomplex1
 		log += fb->scale_log2;
 
 		/* Change Q5.26 Q5.16 Subtract the applied lshift for power spectra
-		 * log2(x * 2^n) = log2(x) - n
+		 * log2(x * 2^n) = log2(x) - n. Note that the bitshift need to be subtracted
```

Comment:
shouldn't that be `2^-n` or `x / 2^n` or `log2(x) + n`?

Reply:
Yep!